### PR TITLE
applications: nrf_desktop: Add Device Description documentation

### PR DESCRIPTION
**Pull request by MarekPieta**
Change adds Device Description (`dev_descr`) module documentation.

### CODE REVIEW

File: applications/nrf_desktop/doc/dev_descr.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.. _dev_descr:
+
+Device Description
+##################
+
+The ``dev_descr`` module defines custom GATT Service, which contains information if the Peripheral supports the LLPM.
```
Comment (greg-fer) [inline applications/nrf_desktop/doc/dev_descr.rst]:
```suggestion
The ``dev_descr`` module defines custom GATT Service, which contains information about whether the peripheral supports the Low Latency Packet Mode (LLPM).
```

File: applications/nrf_desktop/doc/dev_descr.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Only the nrfxlib Link Layer (:option:`CONFIG_BT_LL_NRFXLIB`) supports the LLPM extension.
+The Service is mandatory for all the nRF Desktop Peripherals that should connect to the nRF Desktop Centrals.
```
Comment (greg-fer) [inline applications/nrf_desktop/doc/dev_descr.rst]:
```suggestion
The Service is mandatory for all nRF Desktop peripherals that connect to the nRF Desktop centrals.
```
I understand that you must use it with the peripherals that require the centrals, but correct me if I'm wrong.
Also, let's keep centrals decapitalized.

File: applications/nrf_desktop/doc/dev_descr.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Module Events
+*************
+
+The module does not submit any event nor subscribe for any event.
+
+Configuration
+*************
+
+The module is enabled by default for all the Bluetooth Peripheral devices.
```
Comment (greg-fer) [inline applications/nrf_desktop/doc/dev_descr.rst]:
```suggestion
The module is enabled by default for all Bluetooth peripheral devices.
```

File: applications/nrf_desktop/doc/dev_descr.rst [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+``dev_descr.h`` file contains the UUIDs used by the custom GATT Service.
+The file is located in the ``configuration/common`` directory.
+The UUIDs must be the same for all the Peripheral devices to ensure that the Central will be able to properly read information about the LLPM support.
```
Comment (greg-fer) [inline applications/nrf_desktop/doc/dev_descr.rst]:
```suggestion
The UUIDs must be the same for all peripheral devices to ensure that the central is able to read information about the LLPM support.
```

File: applications/nrf_desktop/doc/dev_descr.rst [hunk 5]
```diff
@@ -0,0 +1,28 @@
+.. _dev_descr:
+
+Device Description
```
Comment (greg-fer) [inline applications/nrf_desktop/doc/dev_descr.rst]:
```suggestion
Device description module
```
Now, most of the module page titles have `module` included, so let's stick to this style.

Comment (MarekPieta) [reply applications/nrf_desktop/doc/dev_descr.rst]: ok, changed

File: applications/nrf_desktop/doc/dev_descr.rst (hunk 2 above)
Comment (MarekPieta) [reply applications/nrf_desktop/doc/dev_descr.rst]: To clarify:
The nRF Destkop peripherals can connect with nRF Desktop centrals or with other centrals (like Android phone, Windows 10 laptop etc.). Only nRF Desktop centrals are reading the description provided by this module. nRF Desktop centrals require mentioned description to check if they can switch to LLPM connection parameters, other centrals do not support the LLPM.

**Pull request merged**